### ISSUE TITLE
About finetune and metric

### ISSUE THREAD

**Issue by 599046587lz** [Yuqifan1117/HalluciDoctor#7]
Nice work!!!
if it is possible to upload code for fine-tune and code for calculating MME and CHAIR metrics?

**Comment by 599046587lz** [comment]
I finetune minigpt4 on detail_23k dataset and LLaVA+_dataset, and the CHAIR metric are 6.5% and 6.4%, indicating that the illusion elimination effect was not significant. My detailed settings are shown in the picture below. Can you help me analyze the reason? If possible, could you please share your code for finetune and for obtaining MME and CHAIR evaluation metrics.Thanks!!!
![2041704344984_ pic](https://github.com/Yuqifan1117/HalluciDoctor/assets/56827890/aa7505c5-7744-48f0-bb7b-c3d0cd434d0d)
![2051704344997_ pic_hd](https://github.com/Yuqifan1117/HalluciDoctor/assets/56827890/96222bb0-0f64-4940-b1c8-7dd1177b6541)

**Comment by Yuqifan1117** [comment]
I replaced the caption in cc_sub_align with our LLaVA+ based on the code of MiniGPT-4 for fine-tuning.
![image](https://github.com/Yuqifan1117/HalluciDoctor/assets/48062034/a7ed13e7-68ee-472a-b1c8-1f5ce5bfcdbf)
Moreover, the CHAIR metric computes in obj/rel/attri levels, and our ground truth annotations integrate all objects in the coco and visual genome, whether you fully include them in the calculation process.
btw, we will upload the metric computing process later.

**Comment by 599046587lz** [comment]
> I replaced the caption in cc_sub_align with our LLaVA+ based on the code of MiniGPT-4 for fine-tuning. ![image](https://private-user-images.githubusercontent.com/48062034/294093984-a7ed13e7-68ee-472a-b1c8-1f5ce5bfcdbf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzNDY2ODEsIm5iZiI6MTcwNDM0NjM4MSwicGF0aCI6Ii80ODA2MjAzNC8yOTQwOTM5ODQtYTdlZDEzZTctNjhlZS00NzJhLWIxYzgtMWY1Y2U1YmZjZGJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA0VDA1MzMwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmNWFlNWJhMjM4YWVjZmJjMTM4OWY0ZWNhZDA5NTZkOTMyMTNiMDIzZDk3MmE5NzAyNDhhNmYxYTAwZmM4MDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SNrNETRknzsdqgJOGuU3F402WjFVTGbDTR3ll-K5HE4) Moreover, the CHAIR metric computes in obj/rel/attri levels, and our ground truth annotations integrate all objects in the coco and visual genome, whether you fully include them in the calculation process. btw, we will upload the metric computing process later.

Thanks for your reply. I want to know that which dataset you choose when you finetune minigpt4,LLaVA-150k or just details_23k?

**Comment by Yuqifan1117** [comment]
we just use details_23k
